source/clear-linux/guides/maintenance/autoproxy.rst

```diff
@@ -3,16 +3,151 @@
 Autoproxy
 #########
 
-.. rst-class:: content-collapse
+Autoproxy is provided to enable |CL-ATTR| to work smoothly behind a
+corporate proxy.
 
-About
-=====
+.. contents::
+   :local:
+   :depth: 1
 
-.. include:: autoproxy_about.txt
+Description
+***********
 
-.. rst-class:: content-collapse
+Autoproxy tries to detect a Proxy Auto-Config (PAC) script and use it to
+automatically resolve the proxy needed for a given connection. With
+Autoproxy, you can use |CL| inside any proxy environment without having to
+manually configure the proxies.
 
-Guide
-=====
+Corporate and private networks can be very complex, needing to restrict and
+control network connections for security reasons. The typical side effects
+are limited or blocked connectivity and requiring manual configuration of
+proxies to perform the most mundane tasks such as cloning a repo or checking
+for updates. With Clear Linux, all of the work is done behind the scenes to
+effortlessly use your network and have connections “just work”.
 
-.. include:: autoproxy_guide.txt
\ No newline at end of file
+This feature removes massive complications in network connectivity due to
+proxy issues. You can automate tasks like unit testing without worrying
+about the proxy not being set and you can remove unset proxies from the
+equation when dealing with network unavailability across systems.
+
+How it works
+************
+
+We designed autoproxy around tools provided by nearly any Linux
+distribution with a few minor additions and modifications. We leveraged the
+DHCP and network information provided from systemd and created a
+PAC-discovery daemon. The daemon uses the information to resolve a URL for a
+PAC file. The daemon then passes the URL into PACrunner\*. PACrunner
+downloads the PAC file and uses the newly implemented Duktape\* engine to
+parse it.
+
+.. figure:: figures/autoproxy_0.png
+   :width: 400px
+
+   Figure 1: Autoproxy Flow
+
+From that point on, any cURL\* or network requests query PACrunner for the
+correct proxy to use. We modified the cURL library to communicate with
+PACrunner over DBus. However, cURL will ignore PACrunner and run normally if
+no PAC file is loaded or if you set any proxies manually. Thus, your
+environment settings are respected and no time is wasted trying to resolve a
+proxy. All these steps happen in the background with no user interaction.
+
+Technical details

```

Comment:
I recommend "Troubleshooting" as a subhead since you provide commands with explanations below.